integration-tests/actions/actions.go

```diff
@@ -230,21 +229,16 @@ func TeardownSuite(
 	optionalTestReporter testreporters.TestReporter, // Optionally pass in a test reporter to log further metrics
 	clients ...blockchain.EVMClient,
 ) error {
-	keepEnvs := os.Getenv("KEEP_ENVIRONMENTS")
-	if keepEnvs == "" {
-		keepEnvs = "NEVER"
-	}
-
 	if err := testreporters.WriteTeardownLogs(t, env, optionalTestReporter); err != nil {
 		return errors.Wrap(err, "Error dumping environment logs, leaving environment running for manual retrieval")
 	}
 	for _, c := range clients {
 		if c != nil && chainlinkNodes != nil && len(chainlinkNodes) > 0 {
 			if err := returnFunds(chainlinkNodes, c); err != nil {
+				fmt.Println(environment.FAILED_FUND_RETURN)
```

Comment:
Is it necessary?

Reply:
Yes this is necessary and propagates a fund return failure out of the remote runner to stop the cleanup from cleaning up the environment.

Reply:
Added a comment to explain the reason for it